bazaarvoice/emodb
# EMO-7632 override host for docker environment

## Github Issue #

-

## What Are We Doing Here?

add possibility to override local host for docker env to use `127.0.0.1` or host machine IP

## How to Test and Verify

1. Check out this PR
2. Run `mvn clean install -e -Ddependency-check.skip=true -DskipTests=true -P docker`
3. Run emodb-web in [docker](https://github.com/bazaarvoice/emodb/tree/master/docker#services-start) - `export EMODB_WEB_DC1_LOCAL_HOST=127.0.0.1; docker-compose up -d emodb-web-dc1`
4. Run gatekeeper against dockerized emodb-web
5. Profit

## Risk

### Level 

`Low`
### Required Testing

`Smoke`

### Risk Summary
This feature won't be used in prod envs. This is only for docker local env.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
